CONTRIBUTING.md

```diff
@@ -6,6 +6,13 @@ before you begin your work.
 
 ## Adding New Components
 
+Before any code is written, open an issue providing the following information:
+
+    * Who's the sponsor for your component. A sponsor is an approver who will be in charge of being the official reviewer of the code. For vendor-specific components, it's good to have a volunteer sponsor. If you can't find one, we'll assign one in a round-robin fashion. For non-vendor specific components, having a sponsor means that your use case has been validated.
```

Comment:
Is the sponsor only responsible for the review or also for subsequent maintenance?

Reply:
Thanks for the callout, i've added a note about the sponsor becoming a code owner as well.